src/sentry/static/sentry/app/views/onboarding/welcome.tsx

```diff
@@ -1,77 +1,130 @@
 import React from 'react';
 import styled from '@emotion/styled';
+import {motion, MotionProps} from 'framer-motion';
 
 import Button from 'app/components/button';
 import {t, tct} from 'app/locale';
-import {Config, Organization} from 'app/types';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
 import {analytics} from 'app/utils/analytics';
-import withConfig from 'app/utils/withConfig';
 import withOrganization from 'app/utils/withOrganization';
 
+import FallingError from './components/fallingError';
+import WelcomeBackground from './components/welcomeBackground';
 import {StepProps} from './types';
 
 const recordAnalyticsOnboardingSkipped = ({organization}: {organization: Organization}) =>
   analytics('onboarding_v2.skipped', {org_id: organization.id});
 
 type Props = StepProps & {
   organization: Organization;
-  config: Config;
 };
 
-const OnboardingWelcome = ({organization, onComplete, config, active}: Props) => {
-  const {user} = config;
+const easterEggText = [
+  t("Wan't to destroy real errors? Get started."),
```

Comment:
```suggestion
  t("Want to destroy real errors? Get started."),
```

unsure if this is a typo

Reply:
LOL, it is but that's funny.